IDE/STM32Cube/STM32_Benchmarks.md

```diff
@@ -182,130 +182,167 @@ CPU: Cortex-M7 at 600 MHz
 AES-192-CBC-dec             16 MiB took 1.000 seconds,   15.527 MiB/s
-AES-256-CBC-enc             16 MiB took 1.000 seconds,   15.698 MiB/s
-AES-256-CBC-dec             16 MiB took 1.000 seconds,   15.527 MiB/s
-AES-128-GCM-enc              1 MiB took 1.012 seconds,    1.037 MiB/s
-AES-128-GCM-dec              1 MiB took 1.012 seconds,    1.037 MiB/s
-AES-192-GCM-enc              1 MiB took 1.008 seconds,    1.041 MiB/s
-AES-192-GCM-dec              1 MiB took 1.012 seconds,    1.037 MiB/s
-AES-256-GCM-enc              1 MiB took 1.016 seconds,    1.033 MiB/s
-AES-256-GCM-dec              1 MiB took 1.016 seconds,    1.033 MiB/s
-AES-128-GCM-enc-no_AAD       1 MiB took 1.004 seconds,    1.046 MiB/s
-AES-128-GCM-dec-no_AAD       1 MiB took 1.000 seconds,    1.050 MiB/s
-AES-192-GCM-enc-no_AAD       1 MiB took 1.000 seconds,    1.050 MiB/s
-AES-192-GCM-dec-no_AAD       1 MiB took 1.019 seconds,    1.054 MiB/s
-AES-256-GCM-enc-no_AAD       1 MiB took 1.004 seconds,    1.046 MiB/s
-AES-256-GCM-dec-no_AAD       1 MiB took 1.008 seconds,    1.041 MiB/s
-GMAC Table 4-bit             2 MiB took 1.000 seconds,    1.716 MiB/s
-CHACHA                      32 MiB took 1.000 seconds,   31.714 MiB/s
-CHA-POLY                    15 MiB took 1.000 seconds,   15.308 MiB/s
-POLY1305                    58 MiB took 1.000 seconds,   57.861 MiB/s
-SHA-256                     88 MiB took 1.000 seconds,   88.062 MiB/s
-HMAC-SHA256                 83 MiB took 1.000 seconds,   83.032 MiB/s
+AES-256-CBC-enc             16 MiB took 1.000 seconds,   15.723 MiB/s
+AES-256-CBC-dec             15 MiB took 1.000 seconds,   15.356 MiB/s
+AES-128-GCM-enc             10 MiB took 1.000 seconds,   10.132 MiB/s
+AES-128-GCM-dec             10 MiB took 1.000 seconds,   10.083 MiB/s
+AES-192-GCM-enc             10 MiB took 1.000 seconds,   10.156 MiB/s
+AES-192-GCM-dec             10 MiB took 1.000 seconds,   10.083 MiB/s
+AES-256-GCM-enc             10 MiB took 1.000 seconds,   10.156 MiB/s
+AES-256-GCM-dec             10 MiB took 1.000 seconds,   10.107 MiB/s
+AES-128-GCM-enc-no_AAD      10 MiB took 1.000 seconds,   10.229 MiB/s
+AES-128-GCM-dec-no_AAD      10 MiB took 1.000 seconds,   10.132 MiB/s
+AES-192-GCM-enc-no_AAD      10 MiB took 1.000 seconds,   10.181 MiB/s
+AES-192-GCM-dec-no_AAD      10 MiB took 1.000 seconds,   10.107 MiB/s
+AES-256-GCM-enc-no_AAD      10 MiB took 1.000 seconds,   10.181 MiB/s
+AES-256-GCM-dec-no_AAD      10 MiB took 1.000 seconds,   10.132 MiB/s
+GMAC Table 4-bit            46 MiB took 1.000 seconds,   45.835 MiB/s
+CHACHA                      32 MiB took 1.000 seconds,   31.519 MiB/s
+CHA-POLY                    15 MiB took 1.000 seconds,   15.259 MiB/s
+POLY1305                    57 MiB took 1.000 seconds,   56.934 MiB/s
+SHA-256                     88 MiB took 1.000 seconds,   88.184 MiB/s
+SHA-384                      6 MiB took 1.000 seconds,    5.835 MiB/s
+SHA-512                      6 MiB took 1.000 seconds,    5.591 MiB/s
```

Comment:
Do we not enable the SHA-512 acceleration for this chip? I'm pretty sure it has it.

Reply:
Fixed.